src/rard/templates/research/partials/section/mentions_section.html

```diff
@@ -0,0 +1,12 @@
+{% load i18n %}
+<section id="mentions">
+    <div class='d-flex justify-content-between mb-3'>
+        <div>
+            <h5>{% trans 'Mentions' %}</h5>
+            <p class="text-muted">This {{obj_name}} was mentioned in the following items</p>
+        </div>
+    </div>
+    {% for mention in object.mentioned_in_list %}
+        <li class="pb-2" style="list-style-type:circle;"><a href="{% url mention.1|add:':detail' mention.0.pk %}">{{ mention.0 }}</a></li>
```

Comment:
If you use the get_absolute_url method, mentioned_in_list can just be a list of objects so no need for worrying about url name etc.
```suggestion
        <li class="pb-2" style="list-style-type:circle;"><a href="{{ mention.get_absolute_url }}">{{ mention }}</a></li>
```